installer/windows/Product.wxs

```diff
@@ -57,9 +57,18 @@
           Account="[SERVICEACCOUNT]"
           Password="[SERVICEPASSWORD]"
           ErrorControl="normal"
-          Arguments="run --config-dir &quot;[INSTALLFOLDER]\&quot; --identity-dir &quot;[IDENTITYDIR]\&quot; --binary-location &quot;[INSTALLFOLDER]\storagenode.exe&quot; --log &quot;[INSTALLFOLDER]\storagenode-updater.log&quot;"
-          />
-        <ServiceControl Id="StoragenodeUpdaterStartService" Start="install" Stop="both" Remove="uninstall" Name="storagenode-updater" Wait="yes" />
+          Arguments="run --config-dir &quot;[INSTALLFOLDER]\&quot; --identity-dir &quot;[IDENTITYDIR]\&quot; --binary-location &quot;[INSTALLFOLDER]\storagenode.exe&quot; --log &quot;[INSTALLFOLDER]\storagenode-updater.log&quot;">
+            <util:ServiceConfig
+              ServiceName="storagenode-updater"
+              ResetPeriodInDays="1"
+              RestartServiceDelayInSeconds="60"
+              FirstFailureActionType="restart"
+              SecondFailureActionType="runCommand"
+              ThirdFailureActionType="runCommand"
```

Comment:
Why two `runCommand` recovery actions?

Reply:
`ThirdFailureActionType` corresponds to `Subsequent failures`, i.e. 3+

Reply:
My point was that in the original ticket we wanted to try to restart twice and then try to recover. This would have us restart once only.

Reply:
Yes. During the course of discussion on this ticket, we switched to only one restart before recovery. I am not sure if this is due to some limitation, or we decided if it is better. @mniewrzal may remember better.

However, it is not critical at all if it would be 1 or 2 restarts before recovery.

Reply:
:+1:

Reply:
We talked that most probably one failure is enough and make no sense to try once again. We may change this if you feel its not enough.